src/app/components/inline-form/index.html

```diff
@@ -0,0 +1,89 @@
+<sky-demo-page
+  pageTitle="Inline form"
+  skyuxModule=" "
+>
+
+<sky-demo-page-summary>
+	<p>The inline form component renders a form in the current view rather than opening a modal.</p>
+</sky-demo-page-summary>
+	
+<sky-demo-page-content sectionHeading="UX guidelines">
+
+<header><div class="sky-section-heading">Usage</div></header>
+	<header><div class="sky-subsection-heading sky-margin-stacked-default">When to use inline forms</div></header>
+	<stache-row>
+    <stache-column screenSmall="6">
+			<ul>
+			<li>The add/edit form is simple in terms of a small # of fields and low complexity of input controls</li>
+			<li>The surrounding content will not distract the user from the add/edit task</li>
+			<li>There is a high likelihood the user would need to add multiple records in succession</li>
```

Comment:
Not a huge deal, but I avoid "There is" and "There are" sentence constructions whenever possible. It usually leads to passive voice. (Although sometimes it can't be avoided.) ... I'd suggest changing this to "Users are highly likely to add multiple records in succession."

Reply:
ok makes sense